src/main/java/db/FruitStorage.java

```diff
@@ -0,0 +1,9 @@
+package db;
+
+import java.util.ArrayList;
+import java.util.List;
+import model.FruitTransaction;
+
+public class FruitStorage {
+    public static final List<FruitTransaction> fruitStore = new ArrayList<>();

```

Comment:
FruitStorage should store Fruits, not FruitTransactions
public static final Map<String, Integer> storage = ...